### ISSUE TITLE
Missing config_v1.json?

### ISSUE THREAD

**Issue by emesoohc2022** [LqNoob/Neural-Homomorphic-Vocoder#1]
Hi there,
    Thanks for open-source this wonderful repo! Could you please share your config_v1.json for train.py? It may help a lot for reproduction experiments. Besides, are there any audio samples? Thanks in advance!

**Comment by LqNoob** [comment]
> Hi there, Thanks for open-source this wonderful repo! Could you please share your config_v1.json for train.py? It may help a lot for reproduction experiments. Besides, are there any audio samples? Thanks in advance!

@emesoohc2022 I apologize for the late response and thank you for your interest in this repository:

1. We have re-uploaded the [config_fmax_v1.json](https://github.com/user-attachments/files/15812586/config_fmax_v1.json) file.
2. As we did not reserve a validation set, we have performed copy synthesis using data from the training set, and the audio files is named [009997_generated_nhv.wav](https://github.com/LqNoob/Neural-Homomorphic-Vocoder/blob/main/009997_generated_nhv.wav), hoping it will be helpful to you.

**Comment by emesoohc2022** [comment]
This helps a lot! Thanks
